src/gluonts/mx/block/quantile_output.py

```diff
@@ -183,103 +183,104 @@ def compute_quantile_weights(self) -> List:
 
-        self.num_quantiles = num_quantiles
-        self.is_iqf = is_iqf
-        with self.name_scope():
-            if self.is_iqf:
-                self.proj_intrcpt = nn.Dense(1, flatten=False)
-                if self.num_quantiles > 1:
-                    self.proj_incrmnt = nn.Dense(
-                        self.num_quantiles - 1,
-                        flatten=False,
-                        activation="relu",
-                    )  # increments between quantile estimates
-            else:
-                self.projection = nn.Dense(
-                    units=self.num_quantiles, flatten=False
-                )
+    def get_loss(self) -> nn.HybridBlock:
+        return QuantileLoss(
+            quantiles=self.quantiles,
+            quantile_weights=self.quantile_weights,
+            is_equal_weights=True,
+        )
 
-    # noinspection PyMethodOverriding,PyPep8Naming
-    def hybrid_forward(self, F, x: Tensor) -> Tuple[Tensor, Tensor, Tensor]:
-        """
+    def get_quantile_proj(self, **kwargs) -> nn.HybridBlock:
+        return nn.Dense(units=self.num_quantiles, flatten=False)
 
-        Parameters
-        ----------
-        F
-            A module that can either refer to the Symbol API or the NDArray
-            API in MXNet.
-        x
-            input tensor
 
-        Returns
-        -------
-        Tensor
-            output of the projection layer
-        """
+class IncrementalDenseLayerProjection(nn.HybridBlock):
+    """
+    A dense layer that outputs non-decreasing values.
+
+    Parameters
+    ----------
+    num_outputs
+        number of outputs of the layer.
+    """
+
+    @validated()
+    def __init__(self, num_outputs: int, **kwargs):
+        super().__init__(**kwargs)
+
+        self.num_outputs = num_outputs
+        with self.name_scope():
+            self.proj_intrcpt = nn.Dense(1, flatten=False)
+            if self.num_outputs > 1:
+                self.proj_incrmnt = nn.Dense(
+                    self.num_outputs - 1,
+                    flatten=False,
+                    activation="relu",
+                )  # increments between quantile estimates
+
+    def hybrid_forward(self, F, x: Tensor) -> Tensor:
         return (
-            (
-                self.proj_intrcpt(x)
-                if self.num_quantiles == 1
-                else (
-                    F.cumsum(
-                        F.concat(
-                            self.proj_intrcpt(x), self.proj_incrmnt(x), dim=-1
-                        ),
-                        axis=3,
-                    )
+            self.proj_intrcpt(x)
+            if self.num_outputs == 1
+            else (
+                F.cumsum(
+                    F.concat(
+                        self.proj_intrcpt(x), self.proj_incrmnt(x), dim=-1
+                    ),
+                    axis=3,
                 )
             )
-            if self.is_iqf
-            else self.projection(x)
```

Comment:
Yes, it looks like this line is missing in `hybrid_forward` for the `QuantileOutput` class.